src/Kafka/Consumer/Types.hs

```diff
@@ -148,53 +163,56 @@ instance Bitraversable ConsumerRecord where
   bitraverse f g r = (\k v -> bimap (const k) (const v) r) <$> f (crKey r) <*> g (crValue r)
   {-# INLINE bitraverse #-}
 
+{-# DEPRECATED crMapKey "Isn't concern of this library. Use 'first'" #-}
```

Comment:
I deprecated those 3 functions as they are aliases to `Bifunctor` module from `base`, what do you think?